docs/smart_scripts.md

```diff
@@ -4,410 +4,50 @@
+| [link][4]             | smallint(5) | unsigned   | PRI | NO   | 0       |       |               |
+| [event_type][5]       | tinyint(3)  | unsigned   |     | NO   | 0       |       |               |
+| [event_phase_mask][6] | smallint(5) | unsigned   |     | NO   | 0       |       |               |
+| [event_chance][7]     | tinyint(3)  | unsigned   |     | NO   | 100     |       |               |
+| [event_flags][8]      | smallint(5) | unsigned   |     | NO   | 0       |       |               |
+| [event_param1][8]     | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [event_param2][8]     | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [event_param3][8]     | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [event_param4][8]     | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [event_param5][8]     | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [action_type][9]      | tinyint(3)  | unsigned   |     | NO   | 0       |       |               |
+| [action_param1][9]    | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [action_param2][9]    | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [action_param3][9]    | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [action_param4][9]    | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [action_param5][9]    | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [action_param6][9]    | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [target_type][10]     | tinyint(3)  | unsigned   |     | NO   | 0       |       |               |
+| [target_param1][10]   | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [target_param2][10]   | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [target_param3][10]   | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [target_param4][10]   | int(10)     | unsigned   |     | NO   | 0       |       |               |
+| [target_x][10]        | float       |            |     | NO   | 0       |       |               |
+| [target_y][10]        | float       |            |     | NO   | 0       |       |               |
+| [target_z][10]        | float       |            |     | NO   | 0       |       |               |
+| [target_o][10]        | float       |            |     | NO   | 0       |       |               |
+| [comment][11]         | text        |            |     | NO   | NULL    |       | Event Comment |
+
+[1]: #entryorguid
+[2]: #source_type
+[3]: #id
+[4]: #link
+[5]: #event_type
+[6]: #event_phase_mask
+[7]: #event_chance
+[8]: #event_flags
+[9]: #action_type
+[10]: #target_type
+[11]: #comment
```

Comment:
and this? O.o

Reply:
if you add (#event_phase_mask) then it offsets everything to the right by 20  characters, so here I did it in a clean way, using pro markdown techniques

Reply:
I have no idea how that will turn out, guess time will tell :P

Reply:
it's perfect, however you can not reuse the link called "1" "2" etc, so if there are two tables, then need to use another number (or use letters or whatever)

Reply:
https://github.com/azerothcore/wiki/blob/a8f1a1707abb14a7874a4cf4dd62779abfc31e1d/docs/smart_scripts.md#entryorguid